samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -112,6 +119,13 @@ func WithSamplingRefreshInterval(samplingRefreshInterval time.Duration) Option {
 	})
 }
 
+// WithSamplingLogger creates a Option that initializes sampling logger.
+func WithSamplingLogger(logger logr.Logger) Option {
```

Comment:
```suggestion
// WithgLogger creates a Option that initializes the logger.
func WithLogger(logger logr.Logger) Option {
```

The logger is not sampling unless the user has configured it to be, but that's beyond the scope of this interface.

Reply:
You're right.